graphql_compiler/schema_generation/utils.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+def amend_custom_scalar_types(schema, scalars):
+    """Amend the serialization, parsing and description of custom scalar types in the schema.
+
+       Information about the fields, (other than the name), of custom GraphQLScalarType objects is
+       lost when GraphQLSchema objects are serialized into strings. This function uses the
+       original type definitions to mutate the schema and reconstructs the fields of custom
```

Comment:
`reconstructs the fields of custom scalar types` seems somewhat misleading. Given that I filed the issue this is closing, I know what this means (the fields of the Python object representation of the scalar type), but a casual reader would think "GraphQL scalars don't have fields, by definition -- that's what object/interface types are for!" and get very confused.

Let's try to make the language more precise and framed in a way that will be helpful to the end user. They don't care about the fields of the Python object, they care about being able to serialize and deserialize their data.